app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -60,6 +60,13 @@ def promote
       respond_for_async :resource => task
     end
 
+    api :PUT, "/content_view_versions/:id/republish_repositories", N_("Forces a republish of the version's repositories' metadata.")
+    param :id, :identifier, :desc => N_("Content view version identifier"), :required => true
+    def republish_repositories
+      task = async_task(::Actions::BulkAction, ::Actions::Katello::Repository::MetadataGenerate, @version.repositories, :force => true)
```

Comment:
After the action is performed, there is nothing shown on the content view's 'History' or 'Tasks'.  Would it be worthwhile to include this new action?  

History is currently covering publish & promotion tasks; therefore, probably not needed there; however, 'Tasks' does capture all of the actions that can be done on a CVV (publish, promote, remove version).